apps/mobile/package.json

```diff
@@ -61,14 +61,15 @@
     "@rnx-kit/metro-config": "1.3.14",
     "@rnx-kit/metro-resolver-symlinks": "0.1.35",
     "@scure/bip32": "1.5.0",
+    "@scure/bip39": "1.4.0",
```

Comment:
Added this for `mnemonicToSeedSync` in `mnemonicToRootNode` but maybe we have a different way on mobile?

Reply:
both of these methods exist in `@leather.io/crypto`